fabtests/unit/getinfo_test.c

```diff
@@ -729,6 +729,43 @@ static int test_caps_regression(char *node, char *service, uint64_t flags,
 	return 0;
 }
 
+/* Check if at least one of FI_LOCAL_COMM and FI_REMOTE_COMM is set */
+static int test_comm_caps(char *node, char *service, uint64_t flags,
+		struct fi_info *base_hints, struct fi_info **info)
+{
+	struct fi_info *test_hints = NULL;
+	uint64_t test_caps[2] = { FI_LOCAL_COMM, FI_REMOTE_COMM };
+	int ret, i;
+
+	if (base_hints) {
+		test_hints = fi_dupinfo(base_hints);
+		if (!test_hints)
+			return -FI_ENOMEM;
+	}
```

Comment:
I think every iteration of the tests does this already and passes in the duplicated hints. Since the hints are a const going into fi_getinfo you should just be able to modify the hints for both cases.

Reply:
you are right. Thanks.